extension/src/lambda.js

```diff
@@ -0,0 +1,27 @@
+const utils = require('./utils')
+const paymentHandler = require('../src/paymentHandler/payment-handler')
+const setup = require('./config/init/ensure-resources')
+
+const logger = utils.getLogger()
+let initialised = false
```

Comment:
That's partly true. When lambdas are run, they do save state for a time, they are just frozen until the next execution. On a "cold start" they have no state, and so will run this code, however on a "warm start" they will still have that state and so this won't be invoked on every run.

However, as I mentioned in another comment on this PR, I don't think it actually makes sense to ensure resources at all within the lambda execution, as if you're doing that at this point, it is already to late. I believe it would be better to force users to create and manage the types themselves if they decide to go with the lambda approach, this could be assisted through automation with terraform.

Reply:
So you will remove this part with `initialised`?

Reply:
Ah I see now, it's already removed.